.gitignore

```diff
@@ -5,3 +5,10 @@
 /package/pkg
 /package/src
 /media/*.pdf
+
+# Tests
+/tests/specs/ds-token-erc20/*-spec.k
+/tests/specs/hkg-erc20/*-spec.k
+/tests/specs/hobby-erc20/*-spec.k
```

Comment:
I think the `/tests/*` stuff was accidentally cherry-picked from the `trufflecon` branch, please confirm I can safely remove them.